app/views/admin/accounts/show.html.haml

```diff
@@ -100,23 +101,24 @@
             %td
               = table_link_to 'vcard', t('admin.accounts.change_role.label'), admin_user_role_path(@account.user) if can?(:change_role, @account.user)
 
-          %tr
-            %th{ rowspan: can?(:create, :email_domain_block) ? 3 : 2 }= t('admin.accounts.email')
-            %td{ rowspan: can?(:create, :email_domain_block) ? 3 : 2 }= @account.user_email
-            %td= table_link_to 'edit', t('admin.accounts.change_email.label'), admin_account_change_email_path(@account.id) if can?(:change_email, @account.user)
-
-          %tr
-            %td= table_link_to 'search', t('admin.accounts.search_same_email_domain'), admin_accounts_path(email: "%@#{@account.user_email.split('@').last}")
-
-          - if can?(:create, :email_domain_block)
+          - if current.role.permissions_as_keys.include?('sensitive_user_data') || current.role.permissions_as_keys.include?('administrator')
```

Comment:
Same regarding the way permissions are handled.

Reply:
Same as above ^^U have a good day.